src/components/Categories.js

```diff
@@ -4,6 +4,8 @@ import NavBar from './NavBar';
 const Books = () => (
```

Comment:
- [ ] [OPTIONAL] It will be better if renamed the component from `Books` to `Categories` to be consistent with the file name and if another developer work with can understand it easily.

Reply:
Alright, I'll do that @omarsalem7